package.json

```diff
@@ -22,11 +22,16 @@
   },
   "homepage": "https://github.com/lulibrary/alma-webhook-handler#readme",
   "devDependencies": {
+    "aws-sdk": "^2.222.1",
```

Comment:
Needs to be production dependency - Tests won't catch issues if production uses a different version.